openmicroscopy/openmicroscopy
# Chgrp not owner

This fixes https://trac.openmicroscopy.org.uk/ome/ticket/12198, https://trac.openmicroscopy.org.uk/ome/ticket/10588 and
https://trac.openmicroscopy.org.uk/ome/ticket/12305 as requested on forums: https://www.openmicroscopy.org/community/viewtopic.php?f=4&t=7484

To test:
 - Login as root (or any Admin).
 - Browse to another user's data (possibly in a group that root is not a member of).
 - Choose data in tree and right-click -> Move to Group...
 - Should see list of groups that the data owner belongs to (not including current group).
 - Click on a group. Now you should see list of containers (same as existing behaviour).
 - Can choose an existing container OR click "New" to create a new container (if moving Dataset/Images/Plates)
 - Click "New..." and enter name... Click OK.
 - Data should be moved to the chosen group and placed in new container.

 Also:
 - Test that the workflow above works when trying to move split MIFs.
 - Test workflow as owner of the data.
 - Chgrp shouldn't be enabled for group owners (only data owners or Admins).
